backend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14.7
+COPY . /app
+WORKDIR /app
+
+RUN go mod download \
+&& go mod verify \
+&& go build -o build/app main.go
+
+CMD ["/app/build/app"]
```

Comment:
Let's add multi stage build so that we don't have to package all dependencies together into production container to reduce production container size: https://github.com/short-d/app-template/blob/master/backend/Dockerfile